app/api/views/api_view.py

```diff
@@ -0,0 +1,21 @@
+from app.serializers.player_serializer import PlayerSerializer
+from app.models import Player, Spectator, Moderator
+
+from django.http import Http404, JsonResponse
+
+import json
+
+def account_info(request):
+    if not request.user.is_authenticated:
+        print("Not Authenticated")
+        raise Http404 
```

Comment:
This actually shoudn't be a 404. This should maybe be a 403 Unauthorized instead.